### DISCUSSION TITLE
[Github] 깃허브(github) 프로필 꾸미기: productive-box token 오류 해결

### DISCUSSION THREAD

**Discussion by pnudpwls1103** [BDD-CLUB/challenge-every-week-posting#20] [4. 기타 글 모음]
최근 이력서를 작성하면서 github 프로필을 꾸미고 있는데 productive-box를 사용하려다가 생긴 오류를 정리했습니다. 
잔디 먹는 뱀을 적용하면서 생긴 오류도 작성하려고 했는데 분량 조절 실패로 다음 게시글 주제로 작성하려고 합니다 ㅎㅎ

[블로그 링크](https://donkeysdevelpment.tistory.com/112)

**Comment by shkisme** [comment]
우와 잔디 먹는 뱀!! 저도 적용해보고 싶네요 ㅎㅎ 다음 게시글 기다리겠습니당!
